package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "linter-gjslint",
   "main": "./lib/init",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
oops. ok. i've never used apm publish. i'll rebase that out. that's why i made it a separate commit.